Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -1384,7 +1384,7 @@ extension BrowserViewController: WKNavigationDelegate {
 
         tab.expireSnackbars()
 
-        if let url = webView.URL where !ErrorPageHelper.isErrorPageURL(url) {
+        if let url = webView.URL where !ErrorPageHelper.isErrorPageURL(url) && url.scheme != "about" {
```

Comment:
There is now an `AboutUtils` struct with a `isAboutHome(url: NSURL)` method that could probably be used here